src/libraries/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj

```diff
@@ -1,10 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <DefineConstants Condition="'$(TargetsUnix)' == 'true'">$(DefineConstants);Unix</DefineConstants>
+    <DefineConstants Condition="'$(TargetsUnixOrBrowser)' == 'true'">$(DefineConstants);Unix</DefineConstants>
```

Comment:
😢 Let's not do this.  Can we factor these ifdefs out into separate source files?